<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>780</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>820</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>860</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>820</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>780</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>860</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>820</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>780</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB9">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>860</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB11">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>820</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB10">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>780</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB12">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>860</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB14">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>820</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB13">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>780</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB15">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>860</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-300</number>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="singleStep">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>740</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Точки для построения кривой Безье</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>780</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>820</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>860</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Z</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>820</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Шаг построения</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="SB17">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>860</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>860</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Угол поворота</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateBrokenLine">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>910</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Построить кривую</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RotateOX">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>910</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Поворот OX</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RotateOY">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>910</y>
      <width>121</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Поворот OY</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SB16">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>820</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.010000000000000</double>
    </property>
    <property name="value">
     <double>0.010000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
